docs/BR.md

```diff
@@ -347,6 +349,8 @@ The binding SHALL use a digital signature algorithm or a cryptographic hash algo
 
 **Root Certificate**: The self-signed Certificate issued by the Root CA to identify itself and to facilitate verification of Certificates issued to its Subordinate CAs.
 
+**Secret Value**: A Random Value intended to be communicated to a third party, where the security of the associated process relies upon that value remaining secret.
```

Comment:
Similarly, I don't think this is an improvement. I think it hinders readability, and the goal of what you're trying to add ("where the security of ...") is an expression of intent more than it is a definitional issue.